pagopa/io-app
# fix(IT Wallet): [SIW-1748] Credentials details screen, accessibility problems

## Short description
This PR fixes an accessibility issue where the screen reader could not correctly recognize the buttons (Back and contextual Help) when the header was set to transparent.


**Warning: This solution has only been tested on Android.**

## List of changes proposed in this pull request
- Added the `headerMode: "float"` option to `ItwPresentationCredentialDetailScreen`

## How to test
To test the fix, you can try removing the added `headerMode: "float"` option and observe how the screen reader behaves with and without it.

Without `headerMode: "float"`
https://github.com/user-attachments/assets/d28d7784-2138-4d9f-9d26-e15f9dcd641d

With `headerMode: "float"`
https://github.com/user-attachments/assets/d204be98-9788-4fd6-843f-5d4a6bdf9a18
